src/serlo-editor/core/sub-document/editor.tsx

```diff
@@ -28,9 +28,9 @@ export function SubDocumentEditor({ id, pluginProps }: SubDocumentProps) {
   const document = useAppSelector((state) => selectDocument(state, id))
 
   const focused = useAppSelector((state) => selectIsFocused(state, id))
-  const [domFocus, setDomFocus] = useState<'focus' | 'focusWithin' | false>(
-    focused ? 'focusWithin' : false
-  )
+  const [domFocus, setDomFocus] = useState<
+    'focus' | 'focusWithin' | 'focusWithinInline' | false
+  >(focused ? 'focusWithin' : false)
```

Comment:
How about making this type an enum, and replacing `false` with a string like `'false'` or `'notFocused'`?

```typescript
enum DomFocus {
  focus = 'focus',
  focusWithin = 'focusWithin',
  focusWithinInline = 'focusWithinInline',
  notFocused = 'notFocused',
}
```

Reply:
Sounds good to me.

Reply:
see https://github.com/serlo/frontend/pull/2766/commits/9869771faf04a5a1f631d5a4f3c1c29f86682823